core/i18n/resources/en.js

```diff
@@ -57,6 +57,7 @@ Do you want to cancel them?`,
     label: 'Label',
     label_plural: 'Labels',
     language: 'Language',
+    leavePageConfirmMessage: 'Data will be lost if you leave the page, are you sure?',
```

Comment:
better to say:
`There are unsaved changes in the form.
By confirming all changes will be lost.
Do you want to proceed?`

Reply:
ok